engine-rest/engine-rest-openapi/src/main/templates/paths/decision-definition/key/{key}/tenant-id/{tenant-id}/xml/get.ftl

```diff
@@ -0,0 +1,64 @@
+{
+  <@lib.endpointInfo
+      id = "getDecisionDefinitionDmnXmlByKeyAndTenant"
+      tag = "Decision Definition"
+      summary = "Get XML By Key and Tenant"
+      desc = "Retrieves the XML of the latest version of the decision definition for tenant" />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "key"
+        location = "path"
+        type = "string"
+        required = true
+        desc = "The key of the decision definition (the latest version thereof)." />
+
+    <@lib.parameter
+        name = "tenant-id"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the tenant the decision definition belongs to." />
+
+
+  ],
+  "responses" : {
+    <@lib.response
+        code = "200"
+        dto = "DecisionDefinitionDiagramDto"
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "Status 200 response",
+                       "description": "Response for GET `/decision-definition/key/aKey/tenant-id/aTenantId/xml`",
+                       "value": {
+                         "id": "aDecisionDefinitionId",
+                         "dmnXml": "<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
```

Comment:
Example XML is not formatted nicely in swagger editor.

Reply:
The formatting depends on the editor + resolution => we won't make adjustments